api/v3/PaymentProcessor.php

```diff
@@ -154,10 +155,15 @@ function _civicrm_api3_payment_processor_pay_spec(&$params) {
     'type' => CRM_Utils_Type::T_INT,
   ];
   $params['amount'] = [
-    'api.required' => TRUE,
```

Comment:
I think TRUE is better  - ie  a boolean (1 will work but...)

Reply:
@eileenmcnaughton Fine by me :-)